.github/workflows/ci.yml

```diff
@@ -108,29 +108,6 @@ jobs:
         echo There are failed visual tests, please inspect test results.
         exit 1
 
-  # used to validate there is no fails for generated docs
-  ci-documentation:
-    name: Documentation
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v2
-
-      - name: Install Reveal dependencies
-        working-directory: viewer
-        run: yarn --immutable
-
-      - name: Build Reveal
-        working-directory: viewer
-        run: yarn build:prod
-
-      - name: Install NPM dependencies
-        working-directory: documentation
-        run: yarn --immutable
-
-      - name: Build docs
-        working-directory: documentation
-        run: yarn run build
-
```

Comment:
Wait, so doc build is not disabled in CI? What does this do then?

Reply:
Yes, it would disable _production builds_ of docs, but `preview docs` will still be built (and they are the same, just without minification enabled).

Reply:
![image](https://user-images.githubusercontent.com/6741854/150762465-8bc8613a-14ee-43e0-8127-c1c5bf9baf62.png)

Reply:
Ah, I see